client/src/main/java/com/paypal/selion/platform/mobile/ios/UIAPicker.java

```diff
@@ -72,10 +72,11 @@ private WebElement findWheelAtIndex(int index) {
     }
 
     private List<String> asList(String allValues) {
+        String tempAllValues = allValues;
```

Comment:
Just declare a short-lived variable where the assignment happens, in the `if` loop